manifests/sharednetwork.pp

```diff
@@ -0,0 +1,22 @@
+# == Define: dhcp::sharednetwork
+#
```

Comment:
Please include documentation for the new resource in [puppet-string](https://puppet.com/docs/puppet/7/puppet_strings.html) format. This will allow us to auto-generate the documentation.

Include at least a `@summary` and two `@params` comments.